optuna/samplers/_nsgaiii/_elite_population_selection_strategy.py

```diff
@@ -103,7 +103,7 @@ def __call__(self, study: Study, population: list[FrozenTrial]) -> list[FrozenTr
 
 def _generate_default_reference_point(
     n_objectives: int, dividing_parameter: int = 3
-) -> np.ndarray[tuple[int, int], np.dtype[np.float64]]:
```

Comment:
Technically speaking, this typing is stricter and correct, but for consistency, I removed this typing.